src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/statusindicator/StatusIndicatorSkin.kt

```diff
@@ -173,7 +254,7 @@ class StatusIndicatorSkin(control: StatusIndicator) : SkinBase<StatusIndicator>(
         bottomInset: Double,
         leftInset: Double
     ): Double {
-        return rightInset + leftInset + 200
+        return rightInset + leftInset + 2000
```

Comment:
why 2000?

Reply:
oh yeah. that's just it's initial prefWidth. The indicator is able to resize to fit its container, i was having problems getting to scale up. by having it be a larger number it can shrink down/ grow up to fit its container. i could replace it with `Double.MAX_VALUE` or look for another solution if you're not comfortable with this

Reply:
Double.MAX_VALUE would be preferable